distmv/general.go

```diff
@@ -6,10 +6,11 @@
 package distmv
 
 var (
-	badQuantile      = "distmv: quantile not between 0 and 1"
-	badReceiver      = "distmv: input slice is not nil or the correct length"
-	badSizeMismatch  = "distmv: size mismatch"
-	badZeroDimension = "distmv: zero dimensional input"
+	badQuantile        = "distmv: quantile not between 0 and 1"
+	badReceiver        = "distmv: input slice is not nil or the correct length"
+	badSizeMismatch    = "distmv: size mismatch"
+	badZeroDimension   = "distmv: zero dimensional input"
+	badNonPosDimension = "distmv: non-positive dimension input"
```

Comment:
"bad" just means that, that the thing was bad without necessarily saying why. So "badSizeMismatch" would either be "badSize" or preferably "sizeMismatch" (since that is the message). The same argument holds here.